packages/usa-checkbox/src/img/checkbox-indeterminate-alt.svg

```diff
@@ -1,10 +1 @@
-<?xml version="1.0" encoding="utf-8"?>
-<!-- Generator: Adobe Illustrator 28.0.0, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
-<svg version="1.1" id="Layer_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px"
-	 viewBox="0 0 64 64" style="enable-background:new 0 0 64 64;" xml:space="preserve">
-<style type="text/css">
-	.st0{fill-rule:evenodd;clip-rule:evenodd;}
-</style>
-<path class="st0" d="M2.9,35.9c0,1.1,0.4,2,1.2,2.8s1.7,1.2,2.8,1.2h7.9h42.3c1.1,0,2-0.4,2.8-1.2s1.2-1.7,1.2-2.8l0,0V28
-	c0-1.1-0.4-2-1.2-2.8S58.2,24,57.1,24H6.9c-1.1,0-2,0.4-2.8,1.2S2.9,26.9,2.9,28V35.9z"/>
-</svg>
+<svg xmlns="http://www.w3.org/2000/svg" xml:space="preserve" viewBox="0 0 64 64"><path d="M2.9 35.9c0 1.1.4 2 1.2 2.8s1.7 1.2 2.8 1.2h50.2c1.1 0 2-.4 2.8-1.2s1.2-1.7 1.2-2.8V28c0-1.1-.4-2-1.2-2.8S58.2 24 57.1 24H6.9c-1.1 0-2 .4-2.8 1.2S2.9 26.9 2.9 28z" style="fill-rule:evenodd;clip-rule:evenodd"/></svg>
```

Comment:
**Note:** When comparing to [correct8.svg](https://github.com/uswds/uswds/blob/ae7129cafeeff2089e9ccfd4b286c50e8d7d0b5f/packages/usa-checkbox/src/img/correct8.svg?short_path=3270fdb#L1) I noticed we still have a style attribute setting the fill where the correct8 has the fill set within the `path` attribute

```scss
<path fill="#171717" fill-rule="evenodd" ...>
```

The SVG doesn't appear to have any regressions but just flagging in case Cypress continues to flag an error

Reply:
@amyleadem can we move those rules out of `style`?

This doesn't match other SVG's.

Reply:
Good catch. 
Replaced `style="fill-rule:evenodd;clip-rule:evenodd"` with `clip-rule="evenodd"` and `fill-rule="evenodd"` in [0f68f96](https://github.com/uswds/uswds/pull/6162/commits/0f68f96a61efae4c112817e36a225e1e8f9e6ff6).